card/src/main/java/com/airwallex/android/card/view/DccActivity.kt

```diff
@@ -39,6 +39,11 @@ class DccActivity : AirwallexActivity() {
         )[DccViewModel::class.java]
     }
 
+    override fun onBackPressed() {
+        super.onBackPressed()
```

Comment:
Is there a reason we should do this? Presumably this will get back to the `AddPaymentMethodActivity`, which will not handle this result.